src/main/java/pl/patrycja/ox/gameflow/FactoryMode.java

```diff
@@ -0,0 +1,35 @@
+package pl.patrycja.ox.gameflow;
+
+import pl.patrycja.ox.ui.UI;
+import pl.patrycja.ox.ui.UIFactory;
+
+class FactoryMode {
+
+    private int inputParametersSize;
+
+    FactoryMode(int inputParametersSize) {
+        this.inputParametersSize = inputParametersSize;
+    }
+
+    //create game settings object and set board size
+    //unbroken line, ui taking amount of args
+    Mode setMode() {
+        switch (inputParametersSize) {
```

Comment:
Try to replace with map. You do not want to adding each time case condition if new mode appear.

Reply:
Ok, I will change it.